apis/upbound/v1alpha1/space_types.go

```diff
@@ -35,6 +35,10 @@ const (
 	// SpaceProviderLabelKey is the key used to identify the cloud provider for
 	// the space. The value should always match the `spec.provider` field.
 	SpaceProviderLabelKey = "spaces.upbound.io/provider"
+
+	// SpaceInaccessibleLabelKey is the key used to indicate that the space
+	// cannot be accessed by the current user.
```

Comment:
```suggestion
	// SpaceInaccessibleLabelKey is the key used to indicate that the current user is
	// not entitled to use the space.
```

'Accessed' implies a permission, which isn't really what this is.